src/services/notifications.service.js

```diff
@@ -470,43 +470,32 @@ const getMainPartnershipContactName = (partners) => {
   return mainPartnershipContact.partner_name;
 };
 
-const transformEstablishmentTypeForNotify = (establishment_type) => {
-  if (establishment_type) {
-    return establishmentTypeEnum[establishment_type].value;
-  }
+const transformEstablishmentTypeForNotify = (establishmentType) => {
+  return establishmentTypeEnum[establishmentType].value;
```

Comment:
What is this correcting? We could do ternary here, right?

Reply:
Added ternary statements in